### ISSUE TITLE
Documentation Needed

### ISSUE THREAD

**Issue by PrabhanshuAttri** [nirmankarta/PyBeacon#2]
http://pythonhosted.org/PyBeacon

**Comment by raarts** [comment]
Return 404 Not Found.

**Comment by vijayant123** [comment]
https://pypi.python.org/pypi/PyBeacon

Working link.

**Comment by raarts** [comment]
Yep. Thanks. (BTW I can't close this issue for some reason. No close button)

**Comment by raarts** [comment]
Yep. Thanks

**Comment by PrabhanshuAttri** [comment]
This is link is for documentation purpose. We are working on the documentation

**Comment by ravaan** [comment]
What kind of documentation are you working on, and how can I contribute to it ?

**Comment by PrabhanshuAttri** [comment]
I haven't started it yet. The document will be pretty basic just like [this](http://doc.pypy.org/en/latest/). It will be hosted [here](http://pythonhosted.org/PyBeacon). Here are some guidelines to follow: [Link](https://wiki.python.org/moin/PyPiDocumentationHosting?action=show&redirect=PyPiHostingDocumentation).

**Comment by PrabhanshuAttri** [comment]
@sigmapie8 would you like to do this?

**Comment by sigmapie8** [comment]
sorry , busy nowdays. I'll take time.

**Comment by mathagician** [comment]
@PrabhanshuAttri I want to start my open source journey with this project. can u please guide me.

**Comment by PrabhanshuAttri** [comment]
@mathagician Sure. Read about it [here](https://wiki.python.org/moin/PyPiDocumentationHosting?action=show&redirect=PyPiHostingDocumentation). Let me know if you are not familiar with git.

**Comment by krishamehta** [comment]
Are you still working on this?  I would like to help.

**Comment by PrabhanshuAttri** [comment]
@krishamehta Awesome. Send a PR. Please include help command from README.md.

**Comment by krishamehta** [comment]
Will do.

**Comment by krishamehta** [comment]
Getting an error which says The Travis CI build failed. What should I do?

**Comment by PrabhanshuAttri** [comment]
@krishamehta You can read about python package structure [here](https://tom-christie.github.io/articles/pypi/). I believe Travis build was not your fault. Just push your code to your forked repo and let us know here. We will help you.

**Comment by krishamehta** [comment]
You just want me to fork the repo first? Sorry I'm new to open source.

**Comment by PrabhanshuAttri** [comment]
Yes, please fork it. No worries. Please go through the following links to learn more:

1. [Hello World](https://guides.github.com/activities/hello-world/)
2. [https://guides.github.com/](https://guides.github.com/)

Must do: [Fun with git](https://try.github.io/levels/1/challenges/1)

**Comment by krishamehta** [comment]
Done forking it.

**Comment by krishamehta** [comment]
What next?

**Comment by PrabhanshuAttri** [comment]
@krishamehta you have created the file for Travis CI. This is a different issue. In documentation, you have to create a directory named docs and add few web pages which will be hosted [here](https://pypi.python.org/pypi/PyBeacon). Also travis.yml doesn't have the correct configurations. Please work on the documentation. #13 is assigned to someone else

**Comment by agdhruv** [comment]
Hello @PrabhanshuAttri 
This seems like a good project to start open source contribution. Could you help me with what I can do?

**Comment by PrabhanshuAttri** [comment]
@agdhruv Please read the description and send a PR.

**Comment by thakkardharmik** [comment]
@PrabhanshuAttri Is this is open? Suggest me what needs to be done in that case.

**Comment by PrabhanshuAttri** [comment]
@thakkardharmik Yes, this is available. Visit [this link](https://wiki.python.org/moin/PyPiDocumentationHosting?action=show&redirect=PyPiHostingDocumentation) to know about Python package documentation guidelines. You can join our slack channel [here](http://nirmankarta.herokuapp.com)

**Comment by PrabhanshuAttri** [comment]
Hi @thakkardharmik Just touching base for a quick update on this. Thanks.

**Comment by akshay-verma** [comment]
@PrabhanshuAttri Please find the PR #58 for this.

**Comment by PrabhanshuAttri** [comment]
Thanks @akshay-verma. This has been pending for a long time. Please check my comments and update the PR so that I can merge it soon. :)
